client/dist/style.css

```diff
@@ -0,0 +1,238 @@
+.backgroundImage {
+  margin-left: -4.4%;
+  margin-right: -4.4%;
+  min-height: 450px;
+  background-position: 100%;
+  background-size: auto 100%;
+  background-repeat: no-repeat;
+  background-color: #051c2b;
+  position: relative;
+
+  z-index: 0;
+}
+
+body {
+  width: 100%;
+  max-width: 1254px;
+  min-width: 768px;
+  margin: auto;
+  font-size: 17px;
+
+  color: aliceblue;
+  line-height: 1.3;
+  background-color: #051c2b;
+}
+
+.blueGradient {
+  background: linear-gradient(90deg,#051c2b 45%,transparent 750px);
+  position: absolute;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  z-index: -1
+
+}
+
+.filmInfo {
+  width: 700px;
+  height: 420px;
+  margin-left: 30px;
+  margin-top: 30px;
+
+  z-index: 2
+}
+
+.contentZone {
+  padding-top: 25px;
+  width: 100%;
+  height: 362px;
+
+}
+
+.movieThumbnail {
+  width: 168px;
+  height: 238px;
+  float: left;
+  padding: 0;
+  position: relative;
+
+}
+
+.movieInfo {
+  width:70%;
+  padding-left: 25px;
+  float: left;
+
+}
+
+.movieTitle {
+  margin-bottom: 8px;
+  padding-right: 15px;
+  padding-right: 15px;
+  font-size: 28px;
+  font-family: Helvetica Neue,Helvetica,Arial,sans-serif;
+}
+
+.generalStats {
+  margin-bottom: 5px;
+
+}
+
+.ratings {
+  margin-bottom: 10px;
+
+  display: flex;
+}
+
+.stat {
+  margin-right: 10px;
+}
+
+.date {
+  line-height: 1.3;
+  font-family: SystemFont,Helvetica Neue,Helvetica,Arial,sans-serif;
+  font-weight: 250;
+  text-transform: capitalize;
+
+}
+
+.important {
+  background-color: #b0bec5;
+  color: #000;
+  font-size: 12px;
+  border-radius: 7px;
+  height: 17px;
+  padding: 0 12px;
+  line-height: 20px;
+  display: inline-block;
+  font-weight: 800;
+  font-family: Helvetica Neue,Helvetica,Arial,sans-serif;
+}
+
+.starzone {
+  margin-right: 27px;
+  height: 13px;
+
+  display: inline-flex;
+  font-size: 100%;
+
+}
+
+.nostars {
+  display: inline-block;
+  width: 73px;
+  height: 13px;
+  background-size: 73px 13px;
+  background-repeat: no-repeat;
+  background-image: url(https://www.vudu.com/bluesteel/images/Star_empty.svg);
+}
+
+.stars {
+  background-image: url(https://www.vudu.com/bluesteel/images/Star_full.svg);
+  background-repeat: no-repeat;
+  height:13px;
+}
+
+.starnum {
+  margin-left: 4px;
+  letter-spacing: .05em;
+  font-size: 13px;
+  line-height: 13px;
+  font-family: Helvetica Neue,Helvetica,Arial,sans-serif;
+  font-weight: 300;
+}
+
+.tomato {
+  background-repeat: no-repeat;
+  background-size: auto 100%;
+  margin-right: 15px;
+  padding-left: 16px;
+  font-size: 13px;
+  height: 13px;
+  display: inline-flex;
+  font-weight: 300;
+  font-family: Helvetica Neue,Helvetica,Arial,sans-serif;
+}
+
+.rtzone {
+  line-height: 13px;
+}
+
+.buttonzone {
+  display: flex;
+  margin-top: 8px;
+  margin-bottom: 15px;
+  width: 448px;
+  float:left;
+  height: 30;
+
+}
+
+.mybutton {
+  width: 33.3%;
```

Comment:
Is this button supposed to scale relative to the width of `window`?

Reply:
its taking up one third of the containing div, which is set to 448px

Reply:
@HBMattoon AH I should’ve checked the component. Nice!